pkg/apps/application-list.jsx

```diff
@@ -86,12 +91,26 @@ class ApplicationRow extends React.Component {
         }
 
         return (
-            <tr onClick={left_click(() => cockpit.location.go(comp.id))}>
-                <td><img src={icon_url(comp.icon)} role="presentation" alt="" /></td>
-                <td>{name}</td>
-                <td>{summary_or_progress}</td>
-                <td>{button}</td>
-            </tr>
+            <DataListItem className="app-list" aria-labelledby={comp.name}>
+                <DataListItemRow>
+                    <DataListItemCells
+                        dataListCells={[
+                            <DataListCell isIcon key="icon">
+                                <img src={icon_url(comp.icon)} role="presentation" alt="" />
+                            </DataListCell>,
+                            <DataListCell width={1} key="app name">
+                                {name}
+                            </DataListCell>,
+                            <DataListCell width={4} key="secondary content">
+                                {summary_or_progress}
+                            </DataListCell>,
+                        ]}
+                    />
+                    <DataListAction aria-labelledby={comp.name} aria-label={_("Actions")}>
```

Comment:
This changes strings... But it seems that this string already exists, so should not break translations.